### PR TITLE
[MODEL-10337] missed pr suggestions

### PR DESCRIPTION
**Pull request by scottp-ml**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
This implements suggestions that were missed in the refactor of custom tasks PR.

### CODE REVIEW

File: task_templates/2_estimators/9_python_calibrator/custom.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 class CustomTask(RegressionEstimatorInterface):
     def fit(self, X: pd.DataFrame, y: pd.Series, **kwargs,) -> None:
 
-        self.multiplier = sum(y) / sum(X[X.columns[0]])
+        self.alibration_coefficient  = sum(y) / sum(X[X.columns[0]])
```
Comment (akshoop) [inline task_templates/2_estimators/9_python_calibrator/custom.py:22]:
```suggestion
        self.calibration_coefficient  = sum(y) / sum(X[X.columns[0]])
```

**Review by akshoop — APPROVED** [review]
👍

**Comment by yakov-g** [comment]
jenkins run test_integration_general test_integration_java_environment test_integration_r_environment

**Comment by yakov-g** [comment]
jenkins run test_integration_general  test_integration_r_environment

**Review by lkanggithub — APPROVED** [review]
LGTM

**Pull request merged**